chalicelib/checks/release_updates_checks.py

```diff
@@ -589,7 +592,9 @@ def data_release_updates(connection, **kwargs):
     static_proj = project_filter if project_filter else ''
     static_scope = 'network members' if kwargs['is_internal'] else 'public'
     static_tag = 'with %s tag' % tag_filter if tag_filter else ''
-    static_content = ' '.join(['All', static_proj, 'data released to', static_scope, 'between', start_date_str, 'and', end_date_str, static_tag])
+    static_content = ' '.join(
```

Comment:
This again might be better done with a string and and named interpolations, but is OK this way also. I worry about this particular style because it's not very tolerant of numeric data creeping in.